Microsoft.Toolkit.Uwp.UI/Triggers/IsNullOrEmptyStateTrigger.cs

```diff
@@ -28,22 +29,47 @@ public object Value
         /// Identifies the <see cref="Value"/> DependencyProperty
         /// </summary>
         public static readonly DependencyProperty ValueProperty =
-            DependencyProperty.Register(nameof(Value), typeof(object), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(true, OnValuePropertyChanged));
+            DependencyProperty.Register(nameof(Value), typeof(object), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(null, OnValuePropertyChanged));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether a trigger is active.
+        /// </summary>
+        public bool IsActive
+        {
+            get => (bool)GetValue(IsActiveProperty);
+            set => SetValue(IsActiveProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="IsActive"/> DependencyProperty
+        /// Allows user to enable the trigger from XAML
+        /// </summary>
+        public static readonly DependencyProperty IsActiveProperty =
+            DependencyProperty.Register(nameof(IsActive), typeof(bool), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(false, OnIsActiveChanged));
+
+        private static void OnIsActiveChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue != null && e.NewValue is bool)
+            {
+                var obj = (IsNullOrEmptyStateTrigger)d;
+                obj.SetActive((bool)e.NewValue);
+            }
+        }
```

Comment:
State triggers doesn't use public IsActive properties, but just the base SetActive method.. It's not clear what purpose this one serves? It seems like you can activate this trigger by either setting it to true or the Value to not-null? Which one takes precedence?

Reply:
When putting in the fix for this issue (#4411), the fix worked _except_ for the initial load of the control/sample page.  It's as if the trigger wasn't being evaluated at load time.  Hence, the need to at least initially set the trigger to be active.  Another hack would have been to just assign Sting.Empty to the SelectedItem in the constructor of the page's code-behind, but I didn't like that approach, and implemented the bool DP instead.

Reply:
Since the default value is `null`, it means it is active at creation, so you should call `SetActive(true)` in the constructor.

Reply:
> Since the default value is null, it means it is active at creation

This logic holds, until it _doesn't_ (trigger is not turning On, as expected), which is a condition that was discovered as part of this PR (see the issue with `SelectedItem`, as mentioned above).

However, by calling `SetActive(true)` in the constructor of the trigger, as suggested, you'd then be giving this trigger an artificial value ("On") by default.  What if a developer does *not* want the trigger to be on by default?  (The logic in the trigger may just wind up overwriting this any way, or it may not.)  You'd then be requiring the developer to find a way to turn the trigger off (or, at least, there would be _ambiguity_ as far as if they would have to shut it off, or not).  

Contrast that whole approach, by just simply giving our developers an easy way to set an initial state for the trigger, by exposing a boolean property.  I've gone a little further, and exposed this boolean as a _dependency_ property, which allows for easy setting of the property in the XAML.  

Another approach was the hack (again, mentioned above), of just setting `SelectedItem` to `String.Empty` in the Sample Page's code-behind- same result (trigger is now turning On initially), but clearly a hack.

Reply:
> However, by calling SetActive(true) in the constructor of the trigger, as suggested, you'd then be giving this trigger an artificial value ("On") by default.

How is that artificial? The Value property is `null`, which means the trigger should be `on`, until the Value property changes to something not-null.

> What if a developer does not want the trigger to be on by default?

Then don't set `Value` to `null`?

Reply:
Maybe the problem is that we don't have an analogous `IsNotNullOrEmptyStateTrigger` like we do with Equal/NotEqual?

In the case this trigger is being used, it should be bound to and either that binding is invalid (which is effectively null) or evaluates to some value. The default state of the trigger being on shouldn't be a problem here between the time the trigger is instantiated, the value property is resolved via binding, and the UI is actually displayed.